apps/frontend/src/app/components/HotelTables/PopupForms/HotelCreateDialog.tsx

```diff
@@ -18,19 +20,34 @@ export const HotelCreateDialog = ({
   setOpenCreate,
 }: HotelCreate) => {
   const [image, setImage] = useState(Object);
+  const { hotelInfoList } = useHotelContext();
+  const checkId = () => {
+    const check = hotelInfoList.map((item) => {
```

Comment:
sorry, in frontend should not check the hotelIdId unique or not, this task should assign to backend